openquake/qa_tests_data/event_based_risk/case_2/expected/rlz-000-structural-event-loss-asset.csv

```diff
@@ -1,23 +1,23 @@
-col=00|ses=0003|src=1|rup=001-01,a0,3.461644E+02
-col=00|ses=0003|src=1|rup=001-02,a0,1.018669E+03
-col=00|ses=0003|src=2|rup=001-01,a1,9.634527E+01
-col=00|ses=0004|src=1|rup=003-01,a0,2.400130E+03
-col=00|ses=0005|src=2|rup=001-01,a1,3.928732E+02
-col=00|ses=0005|src=3|rup=002-01,a2,8.905182E+02
-col=00|ses=0005|src=3|rup=002-01,a3,5.078650E+02
-col=00|ses=0006|src=1|rup=004-01,a0,4.866885E+02
-col=00|ses=0009|src=2|rup=005-01,a1,1.120086E+03
-col=00|ses=0010|src=2|rup=001-01,a1,2.154048E+02
-col=00|ses=0011|src=2|rup=002-01,a1,2.853674E+02
-col=00|ses=0013|src=1|rup=003-01,a0,2.197546E+02
-col=00|ses=0013|src=2|rup=001-01,a1,1.119948E+03
-col=00|ses=0015|src=2|rup=001-01,a1,2.883661E+02
-col=00|ses=0016|src=2|rup=005-01,a1,7.697336E+02
-col=00|ses=0018|src=2|rup=001-01,a1,1.252551E+02
-col=00|ses=0018|src=2|rup=001-02,a1,9.477918E+01
-col=00|ses=0018|src=2|rup=001-03,a1,4.355708E+02
-col=00|ses=0018|src=3|rup=001-01,a2,2.536467E+02
-col=00|ses=0018|src=3|rup=001-01,a3,3.574349E+02
-col=00|ses=0019|src=1|rup=004-01,a0,6.218350E+02
-col=00|ses=0019|src=3|rup=004-01,a2,8.235782E+02
-col=00|ses=0019|src=3|rup=004-01,a3,1.035791E+03
+col=00|ses=0003|src=2|rup=001-01,a1,9.634381E+01,0.000000E+00
```

Comment:
This is right. The last column contains the insured losses and in this test there is no insurance.